ocs_ci/ocs/workload.py

```diff
@@ -9,7 +9,7 @@
 class WorkLoad(object):
     def __init__(
         self, name=None, path=None, work_load=None, storage_type='fs',
-        pod=None
+        pod=None, jobs=1
```

Comment:
Need an entry in the docstring for this new arg.

Reply:
Done